packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -533,18 +569,13 @@ export const forgetCoinjoinAccounts =
                 }
             }
             return res;
-        }, [] as Account['symbol'][]);
```

Comment:
Why not `NetworkSymbol` directly?

Reply:
It's not my code, but that's not important of course. I think it's almost equivalent and with `Account['symbol']` we don't need to import one more type. And I actually like a bit more `Account['symbol']` in this case because theoretically if the type interface of Account changed, we might not need to change it here. But I would be fine with `NetworkSymbol` here too.

Reply:
I was not insisting on it, of course, I am unsure myself it it's better to always refer tho the original type or access a property 🤷‍♂️

Reply:
Good material for the style guide. Contrary to @matejkriz's argument, if using `Account['symbol']` and the symbol property is moved so that it is not a property of `Account` anymore, then it would require a change here. `NetworkSymbol` also looks better. So I support using the type directly if it is defined.